obstacles/seesaw/index.js

```diff
@@ -0,0 +1,5 @@
+const TimerSeesaw = require('./TimerSeesaw');
```

Comment:
Index files indeed look useful. Maybe there's also other places where they could come in handy?